.github/workflows/sponsors_readme.yml

```diff
@@ -1,14 +1,17 @@
 name: Generate Sponsors README
+
 on:
   workflow_dispatch:
   schedule:
     - cron: '0 */12 * * *'
+
 jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout 🛎️
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
```

Comment:
Side note should we add contributors to this or would that make the problem worse

Reply:
Maybe later, but that workflow is disabled atm and idk who is working on it or what it's supposed to do exactly.